node/src/components/consensus/era_supervisor.rs

```diff
@@ -167,6 +170,12 @@ where
         let metrics = ConsensusMetrics::new(registry)
             .expect("failure to setup and register ConsensusMetrics");
         let genesis_start_time = protocol_config.timestamp;
+        let protocol_version = ProtocolVersion::from_parts(
+            protocol_config.protocol_version.major as u32,
+            protocol_config.protocol_version.minor as u32,
+            protocol_config.protocol_version.patch as u32,
+        );
```

Comment:
We should have a `From` impl for this.

Reply:
I agree, and I brought this up in another PR I reviewed.
I think there were concerns about supporting `no-std`… I'd be happy to add this, but maybe in another PR?

Reply:
Of course.